README.md

```diff
@@ -161,14 +161,14 @@ yarn dev-up
 
 You can view your local UI server at http://localhost:4200/.  
 
-By default, this connects to our test API server. Use `--configuration=$ENV` to
+By default, this connects to our test API server. Prepend `REACT_APP_ENVIRONMENT=$ENV` to
```

Comment:
nit: I would reword to something like.. "Prepend..." -> "Set the `REACT_APP_ENVIRONMENT` environment variable"